### ISSUE TITLE
No alt text

### ISSUE THREAD

**Issue by fosterc1** [cgkineo/adapt-graphicLottie#4]
Show the graphics used (svgz) when using this plugin show an alt tag like the jpg/png versions?

Thanks

**Comment by kirsty-hames** [comment]
Hi @fosterc1, this plugin already supports the use of alt tags, just how you would a component graphic or `<img>`. If no alt text is set then `aria-hidden` is applied instead. Does this answer your issue?

**Comment by swashbuck** [comment]
@fosterc1 Just following up. Did the explanation from @kirsty-hames make sense or did you have other concerns? Thanks!

**Comment by oliverfoster** [comment]
closing as stale
